kong/clustering/services/sync/rpc.lua

```diff
@@ -408,6 +421,12 @@ function sync_once_impl(premature, retry_count)
   local current_version = get_current_version()
   if current_version >= latest_notified_version then
     ngx_log(ngx_DEBUG, "version already updated")
+
+    -- version changed, we should update status
+    if version_before_sync ~= current_version then
+      update_status(current_version)
```

Comment:
We could start a timer to call the notification here as it ignores the response , so to release the lock quicker

Reply:
The rpc notification is very cheap, I think that we need not to run a new timer here.

Reply:
i checked the inside logic of notify, there's no block logic, at last, it will call msg push, is it right ? so i think do not need to introduce a timer


final logic of notify()
```
-- asynchronously start executing a RPC, _node_id is not
-- needed for this implementation, but it is important
-- for concentrator socket, so we include it just to keep
-- the signature consistent
function _M:call(node_id, method, params, callback)
  assert(node_id == self.node_id)

  local id

  -- notification has no callback or id
  if callback then
    id = self:_get_next_id()
    self.interest[id] = callback
  end

  return self:push_request({
    jsonrpc = jsonrpc.VERSION,
    method = method,
    params = params,
    id = id,
  })
end
```